external-crates/move/crates/move-compiler-transactional-tests/tests/dependencies/public_package_different_packages.move

```diff
@@ -8,6 +8,7 @@ module 0x42::X {
     }
 }
 
+// TODO: This test may need to be updated when we add package loading
```

Comment:
No way to rewrite it now?

Reply:
Actually sat down and thought about this for a minute -- I don't think this test needs to be changed at all. Since we should error when we try to publish the second module ("package") since when we try to publish the second module ("package") there's an existing package with the same address already published.